src/Orleans.Serialization.Abstractions/Annotations.cs

```diff
@@ -46,6 +46,7 @@ public GenerateMethodSerializersAttribute(Type proxyBase, bool isExtension = fal
         /// <value>true if this instance is extension; otherwise, false.</value>
         public bool IsExtension { get; }
 
+#pragma warning disable 1574 // ValueTask not directly in netstandard2.0
```

Comment:
Should we be adding a reference to the ValueTask package for netstandard2.0?

Reply:
Certainly could and definitely would if there was a "real" dependency on it. As is, not intruding a dependency seems like the better option.

Side note: what are these invokers used for? I wasn't finding any references to them.

Reply:
Good point, it's unused now, so I've deleted it in the latest commit

Reply:
Could you select the option on this PR to allow me to push to your branch?